version.json

```
@@ -1,3 +1,3 @@
 {
-  "version": "v0.0.2"
+  "version": "v0.0.3"
```

Since this fix  is technically a breaking change:
```suggestion
  "version": "v0.1.0"
```